rosidl_generator_py/resource/_msg.py.em

```diff
@@ -506,14 +506,14 @@ bound = 2**nbits
 name = "float"
 bound = 3.402823e+38
 }@
-                 all(val >= -@(bound) and val <= @(bound) for val in value)), \
+                 not any(val < -@(bound) or val > @(bound) for val in value)), \
```

Comment:
Maybe this to allow inf too?

```python3
all((not (value < -bound or value > bound)) or math.isinf(value) for value in array_value)
```

The typical case is all values are valid, so I like `all()` assuming it exits when it finds the first invalid value.